src/lib/containers/FullWidthAlert.tsx

```diff
@@ -56,57 +53,72 @@ function FullWidthAlert(props: FullWidthAlertProps) {
     autoCloseAfterDelayInSeconds,
     variant = 'info',
     transition,
-    isGlobal = true} = props;
+    isGlobal = true,
+  } = props
   const iconContent = getIcon(variant)
-  useEffect(
-    () => {
-      let timer: NodeJS.Timeout
-      if (onClose && autoCloseAfterDelayInSeconds) {
-        timer = setTimeout(onClose, autoCloseAfterDelayInSeconds * 1000)
-      }
-      return () => {
-        if (timer) {
-          clearTimeout(timer);
-        }
+
+  const hasActions =
+    (primaryButtonText && onPrimaryButtonClick) ||
+    (secondaryButtonText && secondaryButtonHref)
```

Comment:
Check to see if there's a primary/secondary button. If not, we apply an additional class name to the component